src/atoms/atomgraph.jl

```diff
@@ -127,9 +79,8 @@ Construct an AtomGraph object from a structure file.
 """
 function AtomGraph(
     input_file_path::String,
-    id::String = "",
+    id::String = input_file_path,
```

Comment:
can we strip the extension from this? `splitext` should do it

Reply:
Resolved in b049e0f.